dlc-manager/src/manager.rs

```diff
@@ -1202,7 +1202,7 @@ where
                 })?;
 
             let (signed_cet, closed_channel) =
-                channel_updater::finalize_unilateral_close_settled_channel(
+                channel_updater::finalize_unilateral_close_established_channel(
```

Comment:
weird that it was named "settled channel" before as we only call this from closing an established channel.

Reply:
Yeah, that was really weird :shrug: